src/main/java/org/apache/accumulo/testing/continuous/ContinuousIngest.java

```diff
@@ -64,6 +64,14 @@ private static boolean pauseEnabled(Properties props) {
     return Boolean.parseBoolean(value);
   }
 
+  private static int getDeleteProbability(Properties props) {
```

Comment:
Did you consider using a floating point value instead of an integer?  Being this is being passed in as a String from props it might not matter - but I think it is more common to be able to specify a probability as a ratio, say 1/10 or 1/2 (or 0.1 or 0.5) - for example [Open Tracing Sampling](https://opentelemetry-python.readthedocs.io/en/latest/sdk/trace.sampling.html)

Reply:
I hadn't considered it but I do like that better than what I current have here. I think I'll change it to a float between 0 and 1 (e.g. 0.4).

Reply:
Addressed in 9e25568